types/src/v0/v0_1/state.rs

```diff
@@ -15,12 +15,11 @@ use std::collections::HashSet;
 pub type BlockMerkleTree = LightWeightSHA3MerkleTree<Commitment<Header>>;
 pub type BlockMerkleCommitment = <BlockMerkleTree as MerkleTreeScheme>::Commitment;
 
-pub type FeeMerkleTree = UniversalMerkleTree<FeeAmount, Sha3Digest, FeeAccount, FEE_MERKLE_TREE_ARITY, Sha3Node>;
+pub type FeeMerkleTree = UniversalMerkleTree<FeeAmount, Sha3Digest, FeeAccount, 256, Sha3Node>;
```

Comment:
I think now it's harder to know what the 256 is for if you just look the code outside an editor. I think it would be fine to keep the constant just for that. Could make it not put though, or just put a comment /* arity */

Reply:
4a157c2c2497b8177a3c53cfeba089428e414332